app/routes/which-business.js

```diff
@@ -2,36 +2,46 @@ const joi = require('joi')
 const ViewModel = require('./models/which-business')
 const getAllOrganisations = require('./models/util-which-business')
 const cache = require('../cache')
+const handler = require('./handler')
 
 module.exports = [{
   method: 'GET',
   path: '/which-business',
   options: {
+    pre: [
+      handler.preHandler('which-business')
+    ],
     handler: async (request, h) => {
+      const journeyItem = request.pre.journeyItem
       const { sbis, applyJourney } = await getAllOrganisations(request)
```

Comment:
I'm not sure if this is part of this PR or not, but I'm not keen on the variable `applyJourney`.  It's not really clear what that actually is referring to.